design/card/index.module.css

```diff
@@ -1,12 +1,10 @@
-:root {
+.container {
```

Comment:
This is breaking change, I am not sure we can live with this. If you use `--card-padding-x`, for example, somewhere outside the card container, you do not get any value.

If you want to get rid you from `:root` consider the pseudoselector being `html` in our case. It's called root to avoid writing a list like `html, svg`, Please, replace `:root` with `html` with a TODO and the link to the bug.

Reply:
> This is breaking change, I am not sure we can live with this. If you use --card-padding-x, for example, somewhere outside the card container, you do not get any value

Isn't it right> I mean that variable should be available only within Card scope or moved to global.css file.

I am afraid [none of pure selectors is allowed](https://github.com/zeit/next.js/issues/10142). We just need to wait till they fix it.

Reply:
It is right, it's how CSS works and how the `design` was designed. I do not want to have a central configuration file for it but it's considerable. I quite like [HiQ](https://jonathanharrell.github.io/hiq/)'s approach.

~If it's not possible, can you duplicate selector with linter disabling and a TODO?~ The current solution is fine then.

Reply:
Well, we need to wait anyway because local hot reloading reorders css files. So maybe it gets fixed along with https://github.com/zeit/next.js/issues/10148. If that's the case I will revert back to `:root` usage.